dashbuilder-client/dashbuilder-navigation-client/src/main/resources/org/dashbuilder/client/navigation/resources/i18n/NavigationConstants.properties

```diff
@@ -37,9 +37,9 @@ navCarouselDragComponentHelp=The navigation group you select will be used to ret
 navTabListDragComponentEmptyError=Please, make sure the configured group contains at least one user created perspective.
 navTilesDragComponentEmptyError=Please, make sure the group configured is not empty.
 navCarouselDragComponentEmptyError=Please, make sure the group configured contains at least one user created perspective.
-navTabListDragComponentRecursivityError=This tab is not displayed to avoid an infinite loop. Please, check this component configuration to avoid recursive references.
-navTilesDragComponentRecursivityError=The content is not displayed to avoid an infinite loop. Please, check this component configuration to avoid recursive references.
-navCarouselDragComponentRecursivityError=This carousel slide is not displayed to avoid an infinite loop. Please, check this component configuration to avoid recursive references.
+navTabListDragComponentDeadlockError=This tab is not displayed to avoid an infinite loop. Please, check this component configuration to avoid deadlocks.
```

Comment:
IMO this error isn't very helpful and can be improved to be more user friendly:
- "deadlock" is computer science lingo and should probably be avoided
- "check this component" - it's not clear what user should check..

We should make sure the error gives user some hint on how to fix the problem. It's hard to think of a way to formulate it, but here's my take on this (maybe you can formulate this better):

"This {tab/carousel slide/content} is not displayed to avoid infinite loop. In general the perspectives in the navigation group you selected for this component must not contain navigation components which refer (directly nor indirectly) to the perspectives in that same group."